tests/lib/tools/suite/tests.invariant/task.yaml

```diff
@@ -1,5 +1,11 @@
 summary: tests.invariant is detecting problems
 
+details: |
+    The invariants are used to verify the state of the system under test is no
+    leaving leftovers.
```

Comment:
The summary refers to "problems", are there problem other than "leaving leftovers"?

Reply:
it is because there are many invariants defined, and those all have the objective to detect there are not leftovers